ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py

```diff
@@ -601,3 +601,47 @@ def test_paddle_roll(
         shifts=shift,
         axis=axis,
     )
+
+
+#gather_nd
+@st.composite
+def _input_and_index(draw):
+    dtype = draw(helpers.get_dtypes("float", full=False))
+    dtype_and_input = draw(helpers.dtype_and_values(dtype=dtype, min_num_dims=1))  # ensure at least one dimension
+    print(dtype_and_input)
+    params_rank = len(dtype_and_input[1].shape)
```

Comment:
Hi! Your implementation is failing on this line. `dtype_and_input[1]` is a list object which does not contain `shape` attribute. Please make sure that your implementation does not produce any errors and pass first. Let me once it is fixed for a final PR review. Thanks!

Reply:
Thanks for the review, but can you help me out on how to alternatively make the condition pass.